src/main/java/jenkins/plugins/git/traits/SparseCheckoutPathsTrait.java

```diff
@@ -25,6 +26,7 @@ public SparseCheckoutPathsTrait(SparseCheckoutPaths extension) {
      * Our {@link hudson.model.Descriptor}
      */
     @Extension
+    @Symbol("gitSparseCheckoutPaths")
```

Comment:
Would prefer that traits end with the suffix 'Trait' so that they don't prevent using that symbol for an extension that has the same name (like this one does).

```suggestion
    @Symbol("gitSparseCheckoutPathsTrait")
```